runtests.py

```diff
@@ -20,6 +20,7 @@ def setup_and_run_tests(test_labels=None):
 
     tags = envlist('ANYMAIL_ONLY_TEST')
     exclude_tags = envlist('ANYMAIL_SKIP_TESTS')
+    print('exclude', exclude_tags)
```

Comment:
Oops, looks like some debugging code crept into your commit

Reply:
Indeed, thanks for catching it!